digiroad2-api-viite/src/main/scala/fi/liikennevirasto/digiroad2/ViiteApi.scala

```diff
@@ -231,18 +232,19 @@ class ViiteApi(val roadLinkService: RoadLinkService, val vVHClient: VVHClient,
     val linkId = params("linkId").toLong
     val currentProjectId = params("currentProjectId").toLong
     time(logger, s"GET request for /roadlinks/project/prefillfromvvh (linkId: $linkId, projectId: $currentProjectId)") {
-      projectService.fetchPreFillFromVVH(linkId, currentProjectId) match {
+      projectService.fetchPreFill(linkId, currentProjectId) match {
         case Right(preFillInfo) =>
           Map("success" -> true, "roadNumber" -> preFillInfo.RoadNumber, "roadPartNumber" -> preFillInfo.RoadPart, "roadName" -> preFillInfo.roadName, "roadNameSource" -> preFillInfo.roadNameSource.value)
         case Left(failureMessage) => Map("success" -> false, "reason" -> failureMessage)
       }
     }
   }
 
-  get("/roadlinks/midpoint/:linkId") {
```

Comment:
Maybe we could keep using linkIds in the UI even though those are generated by us? Or do we want to let the user put uuid / version in the URL the same way the user can now put linkId?

Reply:
Got answer: we need to show the UUID/Version.